package.json

```diff
@@ -10,7 +10,9 @@
     "react": "^17.0.0",
     "react-dom": "^17.0.0",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "^4.0.3"
+    "react-scripts": "^4.0.3",
+    "table-sort-js": "*",
```

Comment:
How can table-sort-js depend upon itself?

Reply:
I had installed table-sort-js globally while testing the actual code somewhere else. Perhaps, that is why it was depending on itself. Have made the neccessary changes!